src/cartesian/Bar.tsx

```diff
@@ -307,6 +307,7 @@ export class Bar extends PureComponent<Props, State> {
             className="recharts-bar-rectangle"
             {...adaptEventsOfChild(this.props, entry, i)}
             key={`rectangle-${i}`} // eslint-disable-line react/no-array-index-key
+            role="img"
```

Comment:
I think this is the only place that role="img" should be used. If there are references for implementing it elsewhere, please let me know.

Reply:
Sorry, just looked it up and my assertion doesn't seem to be correct, see: 

https://tink.uk/accessible-svg-line-graphs/

Still we should be inserting a <title> so it can be more valuable.